src/client/Microsoft.Identity.Client/Platforms/netdesktop/NetDesktopPlatformProxy.cs

```diff
@@ -245,11 +244,6 @@ public override IDeviceAuthManager CreateDeviceAuthManager()
             return new NetDesktopDeviceAuthManager();
         }
 
-        public override IMsalHttpClientFactory CreateDefaultHttpClientFactory()
-        {
-            return new NetDesktopHttpClientFactory();
```

Comment:
Are the associated `NetDesktopHttpClientFactory` and dependencies classes used anywhere? If not, please delete the dead code.